docs/cow-protocol/tutorials/cow-swap/custom-links.md

```diff
@@ -71,6 +71,7 @@ Native and wrapped tokens per chain:
 - Ethereum Mainnet: [`WETH`](https://etherscan.io/address/0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2) - wrapped `ETH`
 - Gnosis chain: [`WXDAI`](https://gnosisscan.io/address/0xe91D153E0b41518A2Ce8Dd3D7944Fa863463a97d) - wrapped `XDAI`
 - Arbitrum one: [`WETH`](https://arbiscan.io/address/0x82af49447d8a07e3bd95bd0d56f35241523fbab1) - wrapped `ETH`
+- Base: [`WETH`](https://basescan.org/address/0x4200000000000000000000000000000000000006) - wrapped `ETH`
```

Comment:
Lol, how come it has this weird address? 

I checked and is the real one, but I don't get it how it was created

Reply:
🤷